requirements.txt

```diff
@@ -1,12 +1,12 @@
-attrs==18.2.0
-boto3==1.12.38
-botocore==1.15.49
+attrs
```

Comment:
We should probably lock these to a version. I've been bit by this before

Reply:
Absolutely!